css/styles.css

```diff
@@ -0,0 +1,707 @@
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1.25rem;
+}
+
+.nav_bar_container {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  gap: 0;
+  display: flex;
+  flex-direction: column;
+  padding: 1.5625rem;
+  background-color: rgba(0, 0, 0, 0.786);
+  backdrop-filter: blur(8px);
+  width: 100%;
+  height: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.main_nav_link {
+  display: flex;
+  flex-direction: column;
+  margin-top: 3.75rem;
+  font-family: var(--ff-lato);
+}
+
+.main_nav_link li {
+  width: 95%;
+  padding: 1.5625rem;
+  border-top: 1px solid rgba(228, 228, 228, 0.295);
+  transition: 0.5 ease;
+}
+
+.main_nav_link:nth-last-child() {
+  border-bottom: 1px solid rgba(228, 228, 228, 0.295);
+}
+
+.sub_nav_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_nav_link:hover {
+  color: var(--lineColor);
+}
+
+.sub_about_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_about_link:hover {
+  color: var(--lineColor);
+}
+
+.sub_home_link {
+  transition: 0.7s ease;
+  color: #fff;
+  margin: 1rem 0;
+  font-size: 1em;
+}
+
+.sub_home_link:hover {
+  color: var(--lineColor);
+}
+
+.menu-sections li:hover {
+  border-top: 1px solid rgba(228, 228, 228, 0.605);
+  border-bottom: 1px solid rgba(228, 228, 228, 0.605);
+}
+
+.hamburger_menu {
+  display: block;
+  cursor: pointer;
+}
+
+.ham_bar {
+  display: block;
+  width: 1.5625rem;
+  height: 0.1875rem;
+  margin: 0.3125rem auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: black;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #fff;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger_menu.active .ham_bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: #fff;
+}
+
+.nav_bar_container.active {
+  left: 0;
+}
+
+.ham_bar-1 {
+  display: block;
+  width: 1.5625rem;
+  height: 0.1875rem;
+  margin: 0.3125rem auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
+}
+
+.top_section_container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4375rem;
+  text-align: left;
+  padding: 1.5625rem 1.5625rem 0 1.5625rem;
+}
+
+.main_section_title {
+  font-family: var(--ff-lato);
+  font-size: clamp(0.9rem, 1rem + 5vw, 2.5625rem);
+  color: var(--lineColor);
+}
+
+.theme {
+  font-family: var(--ff-goose);
+  line-height: 3.4375rem;
+  font-size: clamp(0.5rem, 1rem + 8vw, 2.125rem);
+  font-weight: 700;
+  color: #ec5242de;
+  background-color: var(--mainColor);
+  background-image: url('../capstone-image/background-pattern2.jpg');
+  background-size: cover;
+  background-blend-mode: multiply;
+  background-clip: text;
+  background-position-y: bottom;
+  margin-bottom: 0.4375rem;
+}
```

Comment:
- [ ] It will be better to follow design guidelines in Behance and try to be aligned with as possible, kindly take a look at the following table to see the difference.

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/234954471-f912579d-c86f-4cdf-85b2-24b76ae4677f.png)  | ![image](https://user-images.githubusercontent.com/53402600/234954559-7c72eae8-3e05-4388-af0e-c43b1cc6b0cc.png) |